presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -191,6 +188,194 @@ This table can then be queried in Presto::
 
     SELECT * FROM hive.web.page_view;
 
+
+.. _tuning-pref-hive:
+
+Tuning
+-------
+
+The following configuration properties may have an impact on connector performance:
+
+``hive.assume-canonical-partition-keys``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ * **Type:** ``Boolean``
+ * **Default value:** ``false``
+ * **Description:** Disable optimized metastore partition fetching for non-string partition keys. Setting this property allows to avoid ignoring data with non-canonical partition values.
+
+
+``hive.domain-compaction-threshold``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ * **Type:** ``Integer`` (at least ``1``)
+ * **Default value:** ``100``
+ * **Description:** Maximum number of ranges allowed in a tuple domain without compacting it. Higher value will cause more data fragmentation but allows to use row skipping feature when reading ORC data. Setting this value higher may have large impact on ``IN`` and ``OR`` clauses performance in scenarios making use of row skipping.
```

Comment:
Done.
Changed first sentence to:
Maximum number of ranges/values allowed while reading hive data without compacting it.